pkg/engine/rule/r0002_unexpected_file_access.go

```diff
@@ -58,6 +62,25 @@ func (rule *R0002UnexpectedFileAccess) ProcessEvent(eventType tracing.EventType,
 		return nil
 	}
 
+	shouldIgnoreMounts := fmt.Sprintf("%v", rule.GetParameters()["ignoreMounts"]) == "true"
+
+	if shouldIgnoreMounts {
+		mounts, err := getMountPaths(openEvent.PodName, openEvent.Namespace, openEvent.ContainerID, engineAccess)
```

Comment:
This might also be kept at the rule state, no need to call this function every time